OpenOversight/app/utils.py

```diff
@@ -559,20 +560,18 @@ def find_date_taken(pimage):
     if isinstance(pimage, PngImageFile):
         return None
 
-    if pimage._getexif():
+    exif = hasattr(pimage, '_getexif') and pimage._getexif()
+    if exif:
         # 36867 in the exif tags holds the date and the original image was taken https://www.awaresystems.be/imaging/tiff/tifftags/privateifd/exif.html
-        if 36867 in pimage._getexif():
-            return pimage._getexif()[36867]
+        if 36867 in exif:
+            return exif[36867]
     else:
         return None
 
 
 # https://gist.github.com/ngtvspc/a686dda375df122ba1a5dd8e6654532b
```

Comment:
Is this gist link still accurate? And since this is a 1-line function, does it maybe make sense to bring it back into the other function?

Reply:
Oh yeah, should remove the link. I would like to keep the function call though as it makes it clearer what we're doing right?

Reply:
Actually I guess the call to `clear` `exif` data is pretty understandable lol!